core/src/main/java/org/axonframework/commandhandling/model/AggregateMember.java

```diff
@@ -39,16 +42,41 @@
 
     /**
      * Indicates whether commands should be forwarded to this AggregateMember. Defaults to {@code true}.
+     *
+     * @return a {@code boolean} denoting iff commands should be forwarded, yes or no.
      */
     boolean forwardCommands() default true;
 
     /**
      * Indicates whether events should be forwarded to this AggregateMember. Defaults to {@code true}.
+     *
+     * @return a {@code boolean} denoting if events should be forwarded, yes or no.
+     *
+     * @deprecated in favor of the {@code eventRoutingMode}.
      */
+    @Deprecated
     boolean forwardEvents() default true;
 
     /**
+     * Indicates the routing mode used for events within this entity. Defaults to {@code ForwardingMode.ALL} to allow
+     * all events through.
+     *
+     * @return a {@link org.axonframework.commandhandling.model.ForwardingMode} describing what mode should be used in
+     * forwarding events to this Aggregate Member.
+     */
+    ForwardingMode eventRoutingMode() default ForwardingMode.ALL;
```

Comment:
To me, this looks a bit off, why not use 'ForwardingMode eventForwardingMode'. If you agree, this will have a trickle-down effect, there are quite a few places where the exact string 'ForwardingMode eventRoutingMode' appears.

Reply:
Fair point, adjusted all the `eventRoutingMode` occurrences to `eventForwardingMode`.